fontc/src/args.rs

```diff
@@ -31,6 +31,12 @@ pub struct Args {
     #[arg(long, default_value = "false")]
     pub flatten_components: bool,
 
+    /// Whether a component is allowed to have a non-identity 2x2 transform.
+    ///
+    /// If not, any source component that does will be flattened
+    #[arg(long, default_value = "true", action = ArgAction::Set)]
+    pub allow_component_transforms: bool,
```

Comment:
optional: we could use the same name as the ufo2ft's filter `--decompose-transformed-components` which is already familiar to our user, and have the bool flag be false by default (so we don't need to add a value `--allow-component-transforms=false`).

Reply:
gftools builder also calls this option `decomposeTransformedComponents` https://github.com/googlefonts/gftools/blob/b074d8fbb52631030e6c1f524513a5a2eb7b2d56/docs/gftools-builder/README.md?plain=1#L134